server/src/models/configModel.ts

```diff
@@ -35,6 +35,9 @@ export interface ServerConfigModel {
   waitingSubtitle: string;
   logoUrl: string;
   postCall?: PostCallConfig;
+  cosmosDBConnectionString: string | undefined;
+  cosmosDBEndpoint;
```

Comment:
Good catch! I will add `string` type there.

Reply:
I think the cosmosDB properties should all be optional. For example, if I don't need Cosmos DB (i.e. if I choose post call to be 'None', 'MSForms', or 'Custom'), then I should be able to ignore these Cosmos DB properties.

Then, we should properly handle the case where:
- Post Call is set to OneQuestionPoll
- But can't initialize Cosmos (i.e. because connection string is missing, or endpoint is missing, or db name is missing, etc.)
In that case we should return an error.

Conversely, if Post Call is **not** OneQuestionPoll, then it should be OK for the Cosmos properties to be undefined or empty strings.

Reply:
Also consider making an object for cosmos instead of having a flat structure. Something like:

```
{
    ...
    cosmosDb?: {
        connectionString?: string;
        endpoint? string;
        dbName: string
    }
}
```